scripts/_GameMaker.js

```diff
@@ -185,6 +185,8 @@ var GMS_API = {
     "ds_list_size": ds_list_size,
     "ds_list_find_value": ds_list_find_value,
     "json_encode": json_encode,
+    "json_decode": json_decode,

```

Comment:
Is this intentional?

Reply:
Yes we ended up adding the json_decode the the extension side we need it for Facebook Instant Games